src/Components/SavedCardItem.res

```diff
@@ -59,135 +60,137 @@ let make = (
+                  opacity="20%"
+                  padding="46%"
+                  border="1px solid currentColor"
+                />
+              </div>
+              <div className={`PickerItemIcon mx-3 flex  items-center `}> brandIcon </div>
+              <div className="flex flex-col">
+                <div className="flex items-center gap-4">
+                  {isCard
+                    ? <div className="flex flex-col items-start">
+                        <div> {React.string(paymentItem.card.nickname)} </div>
+                        <div className={`PickerItemLabel flex flex-row gap-3 items-center`}>
+                          <div className="tracking-widest"> {React.string(`****`)} </div>
+                          <div> {React.string(paymentItem.card.last4Digits)} </div>
+                        </div>
+                      </div>
+                    : <div> {React.string(paymentMethodType->Utils.snakeToTitleCase)} </div>}
+                  <RenderIf condition={paymentItem.defaultPaymentMethodSet}>
+                    <Icon
+                      size=18
+                      name="checkmark"
+                      style={ReactDOMStyle.make(~color=themeObj.colorPrimary, ())}
+                    />
+                  </RenderIf>
+                </div>
               </div>
             </div>
-          </RenderIf>
-        </div>
-        <div className="w-full ">
-          <div className="flex flex-col items-start mx-8">
-            <RenderIf condition={isActive && isRenderCvv}>
+            <RenderIf condition={isCard}>
               <div
-                className={`flex flex-row items-start justify-start gap-2`}
+                className={`flex flex-row items-center justify-end gap-3 -mt-1`}
                 style={ReactDOMStyle.make(~fontSize="14px", ~opacity="0.5", ())}>
-                <div className="w-12 mt-6"> {React.string("CVC: ")} </div>
-                <div
-                  className={`flex h mx-4 justify-start w-16 ${isActive
-                      ? "opacity-1 mt-4"
-                      : "opacity-0"}`}>
-                  <PaymentInputField
-                    isValid=isCVCValid
-                    setIsValid=setIsCVCValid
-                    value=cvcNumber
-                    onChange=changeCVCNumber
-                    onBlur=handleCVCBlur
-                    errorString=cvcError
-                    inputFieldClassName="flex justify-start"
-                    paymentType
-                    appearance=config.appearance
-                    type_="tel"
-                    className={`tracking-widest justify-start w-full`}
-                    maxLength=4
-                    inputRef=cvcRef
-                    placeholder="123"
-                    height="2.2rem"
-                  />
+                <div className="flex">
+                  {React.string(
+                    `${paymentItem.card.expiryMonth} / ${paymentItem.card.expiryYear->CardUtils.formatExpiryToTwoDigit}`,
```

Comment:
Could you please create a variable and store these values -  paymentItem.card.expiryMonth and then use since I can see it is being used in more than one place